### PR TITLE
Added steps to deploy catalog app

### PR DESCRIPTION
**Pull request by cloudnautique**

### CODE REVIEW

File: content/rancher/v2.x/en/tasks/projects/_index.md [hunk 1]
```diff
@@ -43,9 +43,19 @@ Coming Soon
 
 ## Launching a Catalog App
 
-Coming Soon
+1. Navigate to the `Cluster/Project` you would like to deploy to.
+1. Select `Catalog Apps` from the top level navigation.
+1. Click `Launch` button.
+1. Select apps `View Now` button that you would like to launch.
+1. Under `Configuration Options` enter a name.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/projects/_index.md]: Is this name what is used for a namespace?

File: content/rancher/v2.x/en/tasks/projects/_index.md [hunk 2]
```diff
@@ -43,9 +43,19 @@ Coming Soon
...
+1. Select `Template Version` from drop down.
+1. If you would like to alter the namespace, click `Customize` otherwise a new namespace with the `Name` setting.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/projects/_index.md]: Where is this section?

File: content/rancher/v2.x/en/tasks/projects/_index.md [hunk 3]
```diff
@@ -43,9 +43,19 @@ Coming Soon
...
+1. Answer additional questions.
+    * If this is a standard Helm chart, then you can add key/value pairs.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/projects/_index.md]: how can i add key/value pairs? is that the edit yaml/upload yaml section?

File: content/rancher/v2.x/en/tasks/projects/_index.md [hunk 4]
```diff
@@ -43,9 +43,19 @@ Coming Soon
...
+1. Click `Launch`
+
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/projects/_index.md]: we should add a note that each catalog app launches into their own namespace. not sure what happens if you use a namespace that already exists, does it launch into the existing namespace?

**Comment by MBishop17** [comment]
This pr is ready for review at https://github.com/MBishop17/docs/tree/catalog

**Pull request closed without merging**